src/controllers/collateralController.js

```diff
@@ -0,0 +1,48 @@
+require('dotenv').config();
+const Web3 = require('web3');
+const Tx = require('ethereumjs-tx');
+import {
+  AssetCollateral,
+  GAS_PRICE,
+} from '../constants';
+
+const ADDRESS = process.env.ADDRESS;
+const ADDRESS_PRIVATE_KEY = Buffer.from(process.env.ADDRESS_PRIVATE_KEY, 'hex');
+
+const web3Collateral = new Web3(new Web3.providers.HttpProvider(`https://ropsten.infura.io/v3/${process.env.INFURA_API_KEY}`));
```

Comment:
I'm unsure this is the most adequate name for this `web3` instance. Nothing seems to be involving collateral, thus making it a bit confusing contextual wise.

Reply:
Ah yea, before we had two instances of web3, hence the specific naming. Fixed.